resources/sdk/purecloudjava/templates/Configuration.mustache

```diff
@@ -20,5 +36,245 @@ public class Configuration {
    */
   public static void setDefaultApiClient(ApiClient apiClient) {
     defaultApiClient = apiClient;
+    autoReloadConfig = defaultApiClient.getAutoReloadConfig();
+    logger = defaultApiClient.getLogger();
+    configFilePath = defaultApiClient.getConfigFilePath();
+    ApiClient.LoggingConfiguration loggingConfiguration = defaultApiClient.getLoggingConfiguration();
+    Configuration configuration = new Configuration();
+    configFileHash = configuration.getConfigHash();
+    configuration.applyConfigFromFile();
+    if (autoReloadConfig) {
+      Runnable runnable = () -> configuration.runConfigChecker();
```

Comment:
You can update the config file watching to the event-driven logic now.

Reply:
This was an accidental change to the purecloudjava Configuration.mustache file.  I rebased all purecloudjava files with master for this pr, so there aren't any purecloudjava changes now.